Gemfile.lock

```diff
@@ -377,7 +381,7 @@ DEPENDENCIES
   zendesk_api
 
 RUBY VERSION
-   ruby 2.6.2p47
+   ruby 2.6.3p62
```

Comment:
Is this going to blowup because of our `.ruby-version` file and whatever ruby version we have in docker?

Reply:
This is a dependabot upgrade, we've not updated the reference here yet.
So shouldn't blow up for anyone